src/pages/InvitationPage.tsx

```diff
@@ -19,12 +19,12 @@ const InvitationPage = () => {
   const { roomId } = useParams();
   const navigate = useNavigate();
   const { isOpen, onClose, onOpen } = useDisclosure();
-  const [storedValue] = useLocalStorage(AT_KEY, {});
   const [, setValue] = useLocalStorage('invitation', {});
+  const token = localStorage.getItem(AT_KEY);
   useDocumentTitle('WuMoㅤ|ㅤ우리들의 모임');
 
   useEffect(() => {
-    if (!storedValue.accessToken) {
+    if (!token) {
```

Comment:
```suggestion
    if (!storedValue) {
```
토큰 분리하면서 수정할 때 제가 손을 못봤네요...
객체로 토큰이 저장되는게 아니라 이제 문자열로 저장되어서 훅을 사용하셔도 괜찮고 이 방법으로 접근하셔도 좋아요!
더 꼼꼼하게 확인했어야했는데..

Reply:
커스텀훅으로 사용하는 게 통일성있어서 좋겠네요!!! 커스텀훅 사용하는 걸로 수정하였습니다👍👍